### PR TITLE
Add biopython dependency

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by natefoo** [comment]
To preinstall NumPy, you can use a prebuild command just as is done with bx-python:

https://github.com/galaxyproject/starforge/blob/master/wheels/build/wheels.yml#L6

**Comment by bgruening** [comment]
`error: invalid command 'bdist_wheel'` -  Duck-Duck-Go tells me that I should install `wheels` at first?

File: wheels/build/wheels.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 ---
 
 packages:
+    biopython:
+        version: 1.66
+        prebuild:
+            wheel: "git clone -b linux-wheels https://github.com/natefoo/pip && cd pip && for py in /python/cp*-`uname -m`; do ${py}/bin/python setup.py install && ${py}/bin/pip install --index-url https://wheels.galaxyproject.org/simple/ numpy || exit 1; done"
```
Comment (natefoo) [inline wheels/build/wheels.yml]: Sorry for the bad info @bgruening, this is outdated for manylinux1 wheels. You should not need to install Galaxy pip, so this line can become:

``` yaml
wheel: "for py in /python/cp*-`uname -m`; do ${py}/bin/pip install --index-url https://wheels.galaxyproject.org/simple/ numpy || exit 1; done"
```

Comment (natefoo) [inline wheels/build/wheels.yml]: A second correction: this works for manylinux1 but /python/cp\* under the OS X image does not include pip, so this should do it:

``` yaml
wheel: "for py in /python/cp*-`uname -m`; do ${py}/bin/easy_install pip; ${py}/bin/pip install --upgrade pip ; ${py}/bin/pip install --index-url https://wheels.galaxyproject.org/simple/ numpy || exit 1; done"
```

We should probably just install pip in the build pythons in the image, so I created #108 for this.

**Comment by natefoo** [comment]
`error: invalid command 'bdist_wheel'` can be fixed by inserting setuptools with:

``` yaml
insert_setuptools: true
```

Under the biopython entry in wheels.yml.

**Comment by natefoo** [comment]
@bgruening here you are!: https://depot.galaxyproject.org/starforge/wheels/build-128/

**Comment by bgruening** [comment]
Ah, this is good to know. The error message was a little bit misleading. Thanks for fixing!

**Comment by natefoo** [comment]
These wheels have been copied to wheels.galaxyproject.org

**Pull request merged**